Insertion sort implementation in Python

sanjaypradeep opened the pull request:
Insertion sort implementation in python, easy to understand and explained how efficient than head and other sorts

Commit:
Testing

Testing

Commit:
Revert "Testing"

This reverts commit de7cbfbe57969e8b8e85924edc11e559082de25c.

Commit:
insertion implementation in python, easy to understand and explained how efficient than head and other sorts

hmangukia commented:
Ask the user to explicitly enter values for sorting. Also, the result is not getting printed.

sanjaypradeep committed:
Addressing comments. 

User has to explicitly enter inputs for sorting the list values, also included a return statement for insertionSort method.
